### PR TITLE
DEV-198 default values for placeholder courses

### PR DESCRIPTION
**Pull request by josephined21**
### Description & Implementation
- default values for placeholder area, credits, tags, & level to save a couple clicks for users
before: 
image

after: 
image

- remove 'none' option for level (now the only options are lower and upper level) because we have an error message if the user selects 'none' for level anyways
before: 
image
image

- made 'none' option lowercase for areas to keep consistent with other placeholder fields

### CODE REVIEW

File: lib/components/popups/course-search/search-results/Placeholder.tsx [hunk 1]
```diff
@@ -342,7 +342,7 @@ const Placeholder: FC<{ addCourse: (plan?: Plan) => void }> = (props) => {
             </div>
           </div>
           <Select
-            options={['None', 'N', 'S', 'H', 'E', 'Q'].map((area: any) => ({
+            options={['none', 'N', 'S', 'H', 'E', 'Q'].map((area: any) => ({
```
Comment (mpark63) [inline lib/components/popups/course-search/search-results/Placeholder.tsx:345]: ok if you have time, i'd double check if in the backend we do use 'None' and not 'none' 

Pretty sure you should keep as None, unless you want to make changes in db and backend 

Screenshot 2023-06-14 at 8 52 30 PM

**Review by mpark63 — CHANGES_REQUESTED** [review]
change area default back to 'None'. otherwise, lgtm!

File: lib/components/popups/course-search/search-results/Placeholder.tsx (hunk 1 above)
Comment (josephined21) [reply lib/components/popups/course-search/search-results/Placeholder.tsx:345]: smoge why inconsistent :( oke will do yes ma'am

Comment (mpark63) [reply lib/components/popups/course-search/search-results/Placeholder.tsx]: hoping you tested + nothing breaks

**Review by mpark63 — APPROVED** [review]
thanks for addressing comment, josepog

**Pull request merged**